CWE/localisation/text.csv

```diff
@@ -13838,16 +13838,10 @@ TAJ_populist;Patriot Party;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
 TAJ_progressive;Libertarian Party;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
 TAJ_socialist;Labour Party;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
 TAKE_SIDE_PENALTY;Prestige penalty for not taking a side: �R$VAL$�!;;Malus auf Prestige, da keine Seite gew�hlt wurde: �R$VAL$�!;;;;;;;;;;;x;;;;;;;;;;;;;;;;
-TAN;Tanzania;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_colonial_government;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_mandate_government;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government1;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_colonial_government_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_mandate_government_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_hms_government1_ADJ;Tanganyikan;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
-TAN_ADJ;Tanzanian;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
+TAN;Tanganyika;;;;;;;;;;;;;x,,,;;;;;;;;;;;;;;;;
```

Comment:
Please don’t switch the localisation. Current name should be the base name. This will solve any fringe issues that might arise.